manager/Makefile

```diff
@@ -1,3 +1,4 @@
+SHELL := /bin/bash
```

Comment:
Please revert. 

There is no need to use pushd. Just use cd  (no need to cd back too, it doesn't affect the next command).

Reply:
Done. Thanks